challenges/FizzBuzzTree/fizz_buzz_tree.py

```diff
@@ -0,0 +1,35 @@
+def fizz_buzz(node):
+    """Helper function for fizzbuzz conditionals."""
+    if node.val % 5 == 0 and node.val % 3 == 0:
+        node.val = 'FizzBuzz'
+        return node.val
+    elif node.val % 3 == 0:
+        node.val = 'Fizz'
+        return node.val
+    elif node.val % 5 == 0:
+        node.val = 'Buzz'
+        return node.val
+    else:
+        return node.val
```

Comment:
Since you are returning the `node.val` regardless of the condition, you should move it from inside your conditional to after it.
```python
if node.val % 5 == 0 and node.val % 3 == 0:
        node.val = 'FizzBuzz'
elif node.val % 3 == 0:
        node.val = 'Fizz'
elif node.val % 5 == 0:
        node.val = 'Buzz'
return node.val
```